src/intervals/intervals.jl

```diff
@@ -126,6 +126,21 @@ function ..(a::T, b::S) where {T, S}
     interval(atomic(Interval{T}, a).lo, atomic(Interval{S}, b).hi)
 end
 
+function ..(a::T, b::Irrational{S}) where {T, S}
+    R = promote_type(T, Irrational{S})
+    interval(atomic(Interval{R}, a).lo, atomic(Interval{R}, b).hi)
```

Comment:
I think we should add a method
`atomic(T, a) = atomic(Interval{T}, a)`.

Also, since this line occurs in each of the three versions, could it be refactored out somehow?

Reply:
The reason to have it like this is because all `atomic` methods are defined with small variations of `atomic(::Type{Interval{T}}, x::S)`; see #114. I can certainly introduce the method you suggest, but maybe it will end up to be too generic.

Reply:
I just checked your suggestion and tests pass (locally) in Julia 0.6.3; if you'd like I add a commit with the suggestion.

Reply:
Yes please go ahead! Maybe the type T should be restricted to <: AbstractFloat for example.